invenio_deposit/tasks.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```

Comment:
But then we wouldn't be calling the indexer defined in the API: https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/api.py#L116